### ISSUE TITLE
Mac focus ring (halo)

### ISSUE THREAD

**Issue by riddle** [necolas/normalize.css#12]
Why do you feel there should be only one focus outline for every visitor of a site using normalize.css?

```
a:focus {
    outline: thin dotted;
}
```

This line will successfully erase native blue focus ring available system-wide on a Mac. Besides, declaring it with a shorthand reuses color from current element, making it inconsistent throughout the site for links and form widgets or different color.
